tests/foreman/sys/test_sat_clone.py

```diff
@@ -0,0 +1,129 @@
+# -*- encoding: utf-8 -*-
+"""Test class for ``satellite clone``
+
+:Requirement: SatClone
+
+:CaseAutomation: notautomated
+
+:CaseLevel: System
+
+:CaseComponent: Clone
+
+:TestType: functional
+
+:CaseImportance: High
+
+:Upstream: No
+
+:Setup:
+    1. Upload manifest
+    2. Enable and sync RH products
+    3. Create content views - publish, promote"
+    4. Create Activation keys
+    5. Create custom products, custom repos ((both yum and puppet), sync,
+    create content views - publish, promote
```

Comment:
Missing indentation here and causing travis to fail, run `make can-i-push` and also `make docs` locally before pushing.